spark/src/main/kotlin/com/adevinta/spark/components/icons/IconToggleButton.kt

```diff
@@ -139,7 +139,7 @@ public fun FilledIconToggleButton(
     onCheckedChange: (Boolean) -> Unit,
     modifier: Modifier = Modifier,
     enabled: Boolean = true,
-    shape: Shape = SparkTheme.shapes.full,
```

Comment:
This api need to use the `Shape` instead of the `ButtonShape` otherwise it'll break usages in consumers

Reply:
Alright, will do